doc/classes/TextServer.xml

```diff
@@ -944,6 +944,21 @@
 				Returns [code]true[/code] if locale is right-to-left.
 			</description>
 		</method>
+		<method name="is_valid_identifier" qualifiers="const">
+			<return type="bool" />
+			<argument index="0" name="string" type="String" />
+			<description>
+				Returns [code]true[/code] is [code]string[/code] is a valid identifier.
+				If the text server supports the [constant FEATURE_UNICODE_IDENTIFIERS] feature, a valid identifier must:
+				- Conform to normalization form C.
```

Comment:
```suggestion
				- Conform to normalization from C.
```
? Or is "form C" a type of form?

Reply:
It's "form C", see https://www.unicode.org/reports/tr15/#Norm_Forms